Source/Core/Core/HW/MMIO.cpp

```diff
@@ -304,6 +304,8 @@ void ReadHandler<T>::ResetMethod(ReadHandlingMethod<T>* method)
 
   struct FuncCreatorVisitor : public ReadHandlingMethodVisitor<T>
   {
+    virtual ~FuncCreatorVisitor() {}
```

Comment:
These should be defaulted like:

```cpp
virtual ~FuncCreatorVisitor() = default;
```

[there's a minor difference](http://clang.llvm.org/extra/clang-tidy/checks/modernize-use-equals-default.html)